docs/qa/setup_local_qa_cops_test.rst

```diff
@@ -0,0 +1,177 @@
+.. _setup-local-qa-cops-testing:
+
+========================================================
+Setting up local qa cops test environs (will be updated)
+========================================================
+
+Objective
+---------
+
+Get familiar with Concourse by setting up the bakery-pipeline locally.
+This is the pipeline which is used under the hood for `Content Output Producer (COPS) <https://cops.openstax.org/>`_
+
+Retrieve Project & Config File
+------------------------------
+
+1. Clone repository containing the pipeline configuration file to your local directory:
+
+.. code-block:: bash
+
+   $ git clone git@github.com:openstax/concourse-pipelines.git
+
+2. Change to bakery directory to retrieve the correct pipeline (there are several other pipelines in this repository):
+
+.. code-block:: bash
+
+   $ cd bakery
+
+3. Check to see if the pipeline.yml exists in the this directory:
+
+.. code-block:: bash
+
+   $ ls
+
+You should see pipeline.yml listed. This file is used to configure the pipeline.
+
+Setup local concourse server
+----------------------------
+
+0. Prerequisites:
+
+- `Docker <https://www.docker.com/>`_
+
+- `fly CLI - Concourse command line tool <https://concourse-ci.org/fly.html>`_
+
+- `wget <https://www.gnu.org/software/wget/>`_
```

Comment:
if assuming OSX is the case you might want to suggest installing wget via homebrew.

`brew install wget`